documentation/smg/17_using_pixel_data.md

```diff
@@ -0,0 +1,394 @@
+# Using Pixel Data
```

Comment:
I would probably say this is more "Pixel Data" than just using it. Especially given you start with a definition of why it's needed.